### PR TITLE
[8]Add Page Navigation to your pages

### PR DESCRIPTION
**Pull request by Fiyin985**
## Task: `Name of given task`
- `Changes you made to given task`

### CODE REVIEW

**Comment by Fiyin985** [comment]
Screenshot (20)

**Comment by Fiyin985** [comment]
Screenshot (21)

File: submissions/fiyingabriel/htmlForms/contact form.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```
Comment (ToluAkin) [inline submissions/fiyingabriel/htmlForms/contact form.html:1]: @Fiyin985 You have created this file in [7] task. No need for it here. You can delete this file for this branch.

File: submissions/fiyingabriel/htmlForms/contactUs.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```
Comment (ToluAkin) [inline submissions/fiyingabriel/htmlForms/contactUs.html:1]: @Fiyin985 This file is not required of you, you should remove it.

File: submissions/fiyingabriel/htmlForms/htmlTask.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE HTML>
```
Comment (ToluAkin) [inline submissions/fiyingabriel/htmlForms/htmlTask.html:1]: This file already exist in your `HTML` folder, you should remove this as well.

File: submissions/fiyingabriel/htmlForms/logIn.html [hunk 4]
```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
```
Comment (ToluAkin) [inline submissions/fiyingabriel/htmlForms/logIn.html:1]: @Fiyin985 This file is not required of you, you should remove it.

File: submissions/fiyingabriel/htmlForms/signUp.html [hunk 5]
```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
```
Comment (ToluAkin) [inline submissions/fiyingabriel/htmlForms/signUp.html:1]: In this file, I don't see where you linked your product page from the html folder, your contact form is not linked on this page and vice versa.

**Comment by Fiyin985** [comment]
Screenshot (23)
Screenshot (22)

File: submissions/fiyingabriel/htmlForms/signUp.html (hunk 5 above)
Comment (ToluAkin) [reply submissions/fiyingabriel/htmlForms/signUp.html:1]: @Fiyin985

File: submissions/fiyingabriel/htmlForms/contact form.html [hunk 6]
```diff
@@ -36,7 +34,7 @@ <h2>JustBuyIt</h2>
   <footer>
     <ul>
      <li><a href="#">About us</a></li>
-     <li><a href="#">Contact us</a></li>
+     <li><a href="file:///C:/Users/Fiyin.TBNRFRAGS/Desktop/Dufuna-CodeCamp20/submissions/fiyingabriel/HTML/htmlTask.html">Home Page</a></li>
```
Comment (ToluAkin) [inline submissions/fiyingabriel/htmlForms/contact form.html:37]: This is not relative path, this is absolute path. Use relative path.

File: submissions/fiyingabriel/htmlForms/signUp.html (hunk 5 above)
Comment (ToluAkin) [reply submissions/fiyingabriel/htmlForms/signUp.html:1]: Where did you link the product page in this file? @Fiyin985

File: submissions/fiyingabriel/htmlForms/signUp.html [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+   <label>Language:</label>
+   <input list="country-list" name="country" id="country1">
+
+   <datalist id="country-list">
+   	    <option value="English"/>
+   	    <option value="Spanish"/>
+   	    <option value="French"/>
+   	    <option value="Mandarin"/>
+   </datalist>  <br><br>
+   <div><h3>Already Have An Account?</h3><a href="#">Log In</a><h3>Back to Home Page</h3><a href="../HTML/htmlTask.html">Click Here!</a><br><br><br><a href="contact form.html">Contact Us</a></div>
```
Comment (ToluAkin) [inline submissions/fiyingabriel/htmlForms/signUp.html:27]: You need to link the contact form in the signup page @Fiyin985

**Comment by ToluAkin** [comment]
@Fiyin985 on the product page, you need to link the contact form page and signup page.

**Pull request merged**